BoxContentSDK/BoxContentSDK/OAuth2/BOXAuthorizationViewController.m

```diff
@@ -149,7 +150,8 @@ - (void)viewDidLoad
     } else if ([self.SDKClient.session isKindOfClass:[BOXAppUserSession class]]) {
         __weak BOXAuthorizationViewController *me = self;
         [self.SDKClient autheticateAppUserWithCompletionBlock:^(BOXUser *user, NSError *error) {
-            self.completionBlock(me, user, error);
+            [me.URLSession invalidateAndCancel];
```

Comment:
One alternate approach I was considering is creating a 'close/dismiss' method for the auth view controller, or even a 'pre-close/dismiss cleanup' method (since we don't control how it is presented) which the caller would have to use. This is more likely to work well for all 3rd parties without changes, but this has the issue of someone dismissing this VC by some outside mechanism (not driven by the auth VC). One possibility would be to have both - this approach, plus a pre-dismiss method that the caller can use before externally doing away with this VC. What do you guys think?